docs/en/latest/plugins/jwe-decrypt.md

```diff
@@ -0,0 +1,194 @@
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `jwe-decrypt` Plugin is used to decrypt [JWE](https://datatracker.ietf.org/doc/html/rfc7516) authentication header to a [Service](../terminology/service.md) or a [Route](../terminology/route.md).
+
+A [Consumer](../terminology/consumer.md) of the service then needs to provide a key decrypt the request.
+
+## Attributes
+
+For Consumer:
+
+| Name          | Type    | Required                                              | Default | Valid values                | Description                                                                                                                                                                                 |
+|---------------|---------|-------------------------------------------------------|---------|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| key           | string  | True                                                  |         |                             | Unique key for a Consumer.                                                                                                                                                                  |
+| secret        | string  | True                                                 |         |                             | The decrypt key. This field supports saving the value in Secret Manager using the [APISIX Secret](../terminology/secret.md) resource.       |
+| base64_secret | boolean | False                                                 | false   |                             | Set to true if the secret is base64 encoded.                                                                                                                                                |
+
+For Route:
+
+| Name   | Type   | Required | Default       | Description                                                         |
+|--------|--------|----------|---------------|---------------------------------------------------------------------|
+| header | string | False    | authorization | The header to get the token from.                                   |
+| forward_header | string | False     | authorization  | Set the header name pass the plaintext to the Upstream.   |
+
+## API
+
+This Plugin adds `/apisix/plugin/jwe/encrypt` as an endpoint.
+
+:::note
+
+You may need to use the [public-api](public-api.md) plugin to expose this endpoint.
```

Comment:
Not "you may", you do.

I think this info can be moved out of the admonition and just append it to the sentence above.

Reply:
@kayx23 You can modify the document directly on this PR:) 
looks @fishioon is busy now.

Reply:
@moonming ok will do